crates/shared/src/types/epoch_parameters.rs

```diff
@@ -36,6 +37,15 @@ impl EpochParameters {
     }
 }
 
+impl From<CCProofId> for EpochParameters {
```

Comment:
How do you use this change in the patch b/c this looks unrelated.

Reply:
I use it inside `validate_proofs` to create `EpochParameres` in one-line from `CCProofId`